src/Components/MeditationTimer.js

```diff
@@ -1,16 +1,63 @@
 import React, { useRef, useState, useEffect } from "react";
 import { View, Text, StyleSheet, ImageBackground, Dimensions, Animated, TouchableOpacity } from "react-native";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { GET_MEDITATION_QUERY } from "../queries";
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { useMutation } from "@apollo/client";
+
+// const client = new ApolloClient({
+//   cache: new InMemoryCache(),
+// });
```

Comment:
Totally fine in here and during this project - but for some context for the future: 

You won't want to include any commented out code in your PR unless the team agrees or the lead agrees. Most teams will only allow production-ready code to be PRed. This makes it so that there's less of a chance of random, unused things to be added into the final product. When you're in a big codebase, small unused things start to add up. You can think of it like leaving your used plate out after eating a meal - once in a while is fine but if you do it every day, then they add up quickly and it becomes messy quickly.